### DISCUSSION TITLE
Adjusting Image size/layout within the PDF (static/downloads/output.pdf)

### DISCUSSION THREAD

**Discussion by anniemcewen** [thegetty/quire#395] [Get Help] [answered]
Hello everyone! 

I have a question about how to adjust the size/layout of images in the automatically produced catalog PDF using the "quire pdf" command and Prince XML. 

In our catalog, we have a visual/comic essay. On the website I have created a single page for this essay and inserted JPegs of each panel as simple figures like so:

---
title: "Mouthfail: A Comix Essay by Ganzeer" 
type: essay
weight: 22
pdf:   
---
{{< q-figure src="Ganzeer_Mouthfail_1.jpg" >}}
{{< q-figure src="Ganzeer_Mouthfail_2.jpg" >}}
{{< q-figure src="Ganzeer_Mouthfail_3.jpg" >}}
{{< q-figure src="Ganzeer_Mouthfail_4.jpg" >}}
{{< q-figure src="Ganzeer_Mouthfail_5.jpg" >}}

The format looks great online, but when I run "quire pdf" in Terminal the images are placed in a 4X4 grid on each page, which is too small (see screenshot). Ideally I would like them to sit 2x2, but a single image per page would work as well. 

<img width="542" alt="Screen Shot 2022-07-19 at 2 12 14 PM" src="https://user-images.githubusercontent.com/86244205/179820019-21663796-3881-45c5-8ae1-69b2e2a913c7.png">

Any advice on how to adjust figure sizes or layout in the PDF? Thanks so much!

P.S. The first version of our Quire project is now live! Check it out here: https://carlos.emory.edu/publications/AndIMustScream/

**Comment by geealbers** [comment, marked as answer]
Hi @anniemcewen, you can do this by adding a class to the figure shortcodes and then a corresponding CSS style rule to your `static/css/custom.css` file.

For example, if you add a class called `pdf-two-column` to the shortcodes, like:

```
{{< q-figure src="Ganzeer_Mouthfail_1.jpg" class="pdf-two-column" >}}
```
You can then apply a style rule aimed at that class, and tell it to span all the columns on the page by saving this into your custom.css file:

```css
.content figure.quire-figure.pdf-two-column {
    column-span: all;
 }
```
The class name can be anything you want as long as it's the same in both places. You may also need to adjust the margins or other attributes to get the figures to sit right on the page. And if you're not familiar with CSS, the extra stuff I put in there (`.content figure.quire-figure.`) was to make the selector specific enough to apply the rule. I grabbed this from one of our recent books, but it's possible `.pdf-two-column { column-span: all; }` alone might work.

**Reply by Erin-Cecele** [reply]
Hi @anniemcewen, I wanted to see if Greg's response above answered your question. Thank you!
